autoware_launch/launch/autoware.launch.xml

```diff
@@ -2,8 +2,8 @@
 <launch>
   <!-- Essential parameters -->
   <arg name="map_path" description="point cloud and lanelet2 map directory path"/>
-  <arg name="vehicle_model" default="sample_vehicle" description="vehicle model name"/>
-  <arg name="sensor_model" default="sample_sensor_kit" description="sensor model name"/>
```

Comment:
These lines revert https://github.com/autowarefoundation/autoware_launch/pull/768. 
 Could you please remove these changes?

Reply:
Fixed: https://github.com/autowarefoundation/autoware_launch/pull/806/commits/9f988d86fa49b9eca4109b321cc3420116406f17